index.js

```diff
@@ -42,15 +51,25 @@ class TemplateImportProcessor extends BroccoliFilter {
     let imports = [];
     let rewrittenContents = contents.replace(IMPORT_PATTERN, (_, localName, importPath) => {
       if (importPath.startsWith('.')) {
-        importPath = path.resolve(relativePath, '..', importPath);
-        importPath = path.relative(this.options.root, importPath);
+        importPath = path.resolve(relativePath, '..', importPath).split(path.sep).join('/');
```

Comment:
this is `windows` fix, path.resolve return on windows something like `foo\\bar\\baz`